### PR TITLE
Merge from live tool

### PR DESCRIPTION
**Pull request by Matthewrbowker**
This is a step in moving to a git-based workflow.

Would someone review and merge please?

### CODE REVIEW

File: modules/WebTool-backup.php [hunk 1]
```diff
@@ -0,0 +1,1410 @@
+<?php
```
Comment (MusikAnimal) [inline modules/WebTool-backup.php]: we probably don't need to merge in any backup files (I believe there are several)

**Comment by MusikAnimal** [comment]
the `core` files are memory dumps, they can be added to .gitignore

**Comment by MusikAnimal** [comment]
And are we sure we want to delete the flag images (used in range contribs)? Do they live somewhere else?

File: modules/WebTool-backup.php (hunk 1 above)
Comment (Matthewrbowker) [reply modules/WebTool-backup.php]: Whoops.  I'll generate another commit that removes any backup files.

**Comment by Matthewrbowker** [comment]
Yes, the flag images are actually a simlink to another location on Tool Labs.  I'd rather not remove it, but Git is too smart for that.

I'll remove the core files too, and add them to .gitignore.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+/public_html/wikihistory/enwiki/core
+/public_html/rfap/core
+/public_html/rangecontribs/core
+/public_html/pcount/core
+/public_html/pages/core
+/public_html/editsummary/core
+/public_html/echo/core
+/public_html/ec/core
+/public_html/autoblock/core
+/public_html/core
```
Comment (MusikAnimal) [inline .gitignore:14]: you can probably just add `core` and it will ignore all core files, but this is fine

**Comment by MusikAnimal** [comment]
So a little confused... with this PR we are removing the symlink flag files? I would either (a) correct the references to the files so we don't need symlinks or (b) add the symlinks to gitignore too (which is not idea, methinks). If the symlinks are needed, when we pull this on Tool Labs it will delete them, no?

Also I can't even see all 690 files because the diff is too big ;) Maybe if we do something with those symlinks I can see more of it? I guess I should pull this in on my local so I can see everything

File: .gitignore (hunk 2 above)
Comment (Matthewrbowker) [reply .gitignore:14]: I've always had trouble with regular expressions on a .gitignore file.  You're welcome to correct it :)

**Comment by Matthewrbowker** [comment]
No... it just means the simlink (Which is to /shared/flags/ - outside of xTools' domain) will not be tracked by Git.  Git should leave it alone when we do a pull on tool labs proper.

**Pull request closed without merging**